package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "fast-decode-uri-component": "^1.0.1",
+    "fast-deep-equal": "^3.1.3",
```

Comment:
where is this called?

Reply:
Used for checking if duplicate constraints are passed for the same path. It used to be that you could just check `version == handler.version`, but now that there can be more than one constraint, it has to be some by-value comparison. Happy to vendor an implementation of object equality by value, but I couldn't think of a way to avoid the check so I figured i'd use someone else's maintained code.